bin/generate-gutenboarding-design-thumbnails.js

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env node
+/**
+ * Generate static design screenshot thumbnails for Gutenboarding
+ * Until we can generate tall screenshots on the fly with mshots:
+ * https://github.com/Automattic/mShots/issues/16
+ * https://github.com/Automattic/wp-calypso/issues/40564
+ *
+ * You'll need capture-website and sharp:
+ *
+ * npm i capture-website sharp
+ *
+ * (I didn't want to add these to dependencies just for this script since these are kinda heavy ones)
+ * https://www.npmjs.com/package/capture-website
+ * https://www.npmjs.com/package/sharp
+ */
+
+const captureWebsite = require( 'capture-website' );
+const designs = require( '../client/landing/gutenboarding/available-designs.json' );
+const sharp = require( 'sharp' );
+const wpUrl = require( '@wordpress/url' );
+
+const screenshotsPath = './static/page-templates/design-screenshots'; // Folder to store output images
+const captureMaxHeight = 2200; // Cap long pages to  this pixel height when capturing
+const outputWidth = 600; // Outputted file width
+const viewportHeight = 800; // Browser height for capturing the screenshot
+const viewportScaleFactor = 2; // Browser pixel density for capturing the screenshot
+const viewportWidth = 1280; // Browser width for capturing the screenshot
+
+const getDesignUrl = design => {
+	return wpUrl.addQueryArgs( design.src, {
+		font_headings: design.fonts[ 0 ],
+		font_base: design.fonts[ 1 ],
+	} );
+};
+
+async function run() {
+	await Promise.all(
+		designs.featured.map( async design => {
+			const url = getDesignUrl( design );
+			const file = `${ screenshotsPath }/${ design.slug }_${ design.template }_${ design.theme }.jpg`;
+
+			// Fix `reynolds_rockfield2_rockfield.jpg` first section becoming super tall
+			// @TODO: fix at the source since this will be an issue with mshots API, too.
+			const styles =
```

Comment:
Can be completely removed since the template is going away: https://github.com/Automattic/wp-calypso/issues/40700

Reply:
Nope — I mixed templates. This one is actually not going away. :-)